modules/EnsEMBL/Web/Configuration/Transcript.pm

```diff
@@ -92,10 +92,15 @@ sub populate_tree {
     { 'availability' => 'either database:variation has_variations translation', 'concise' => 'Variants' }
   ));
 
-  $prot_menu->append($self->create_node('PDB', '3D Protein model',
+  $prot_menu->append($self->create_node('PDB', 'PDB 3D protein model',
     [qw( alignment EnsEMBL::Web::Component::Transcript::PDB )],
     { 'availability' => 'transcript translation has_pdbe','concise' => '3D Protein model (PDBe)' }
```

Comment:
Does the value of `concise` need to match the label on l95?

Reply:
IIRC originally the "concise" parameter was there so that if a page has a long title, we can put a shorter one in the lefthand menu, and it is thus optional. However since James (former web team leader who developed most of our code) refused to write any documentation, its purpose got forgotten and people tended to include it out of "superstition". I have no idea if it's still being used correctly in our code :(